lib/tasks/setup_s3.rake

```diff
@@ -0,0 +1,132 @@
+namespace :heroku do
```

Comment:
Scrips should not ask user for any parameters, scripts should require them to be passed as parameters for the script.